bigchaindb/core.py

```diff
@@ -429,37 +440,37 @@ def write_vote(self, block, vote, block_number):
         if 'block_number' not in block:
             update['block_number'] = block_number
 
-        r.table('bigchain')\
-         .get(vote['vote']['voting_for_block'])\
-         .update(update)\
-         .run(self.conn)
+        r.table('bigchain') \
+            .get(vote['vote']['voting_for_block']) \
+            .update(update) \
+            .run(self.conn)
 
     def get_last_voted_block(self):
         """Returns the last block that this node voted on."""
 
         # query bigchain for all blocks this node is a voter but didn't voted on
-        last_voted = r.table('bigchain')\
-            .filter(r.row['block']['voters'].contains(self.me))\
-            .filter(lambda doc: doc['votes'].contains(lambda vote: vote['node_pubkey'] == self.me))\
-            .order_by(r.desc('block_number'))\
-            .limit(1)\
+        last_voted = r.table('bigchain') \
+            .filter(r.row['block']['voters'].contains(self.me)) \
+            .filter(lambda doc: doc['votes'].contains(lambda vote: vote['node_pubkey'] == self.me)) \
+            .order_by(r.desc('block_number')) \
+            .limit(1) \
             .run(self.conn)
 
         # return last vote if last vote exists else return Genesis block
         last_voted = list(last_voted)
         if not last_voted:
             return list(r.table('bigchain')
-                         .filter(r.row['block_number'] == 0)
-                         .run(self.conn))[0]
+                        .filter(r.row['block_number'] == 0)
+                        .run(self.conn))[0]
 
         return last_voted[0]
 
     def get_unvoted_blocks(self):
         """Return all the blocks that has not been voted by this node."""
 
-        unvoted = r.table('bigchain')\
-            .filter(lambda doc: doc['votes'].contains(lambda vote: vote['node_pubkey'] == self.me).not_())\
-            .order_by(r.asc((r.row['block']['timestamp'])))\
+        unvoted = r.table('bigchain') \
+            .filter(lambda doc: doc['votes'].contains(lambda vote: vote['node_pubkey'] == self.me).not_()) \
```

Comment:
I understand using `r.row` to eliminate one `lambda` function but why move away from the python syntax when `eq` an `==` are the same?